packages/app/index.js

```diff
@@ -1,10 +1,35 @@
+const dotenv = require("dotenv");
+require("dotenv").config();
```

Comment:
Why  we have this here ?

Reply:
1. I have loaded the environment vars at the top so that it is accessible at time of loading newrelic. Removed line 11 as it is no longer required.

2. Also made sure that by default if no `NODE_ENV` is declared, "dev" is treated as the default `NODE_ENV`.